### PR TITLE
Update Taxonomy support in Get List API

### PR DESCRIPTION
**Pull request by Vridhi-Chowdhry**
Update Taxonomy support in Get List API

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97f4a55](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/pull/3846/commits/97f4a55fab366d0373743714febea608c7a1eb6d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97f4a55](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/pull/3846/commits/97f4a55fab366d0373743714febea608c7a1eb6d)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97f4a55](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/pull/3846/commits/97f4a55fab366d0373743714febea608c7a1eb6d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/list-get.md](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/blob/patch-1/api-reference/beta/api/list-get.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Flist-get.md)

<a name="api-reference%2Fbeta%2Fapi%2Flist-get.md"></a>
### [api-reference/beta/api/list-get.md](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/blob/patch-1/api-reference/beta/api/list-get.md)
 - **Line 193**: **[Warning]** Invalid file link:(~/beta/api/ManagedMetadata.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901240538044790-3846%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cTKU8%2BMrdMaHFE2jYCQbanwHZ%2Ftwbp%2FIInvCdD9SA8E%3D&st=2019-01-24T05%3A36%3A25Z&se=2019-02-24T05%3A41%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/list-get.md [hunk 1]
```diff
@@ -129,6 +129,69 @@ Content-type: application/json
 }
 ```
 
+#### Request
```
Comment (Lauragra) [inline api-reference/beta/api/list-get.md]: Please organize each of the examples in this file under H3s with descriptions, i.e. "### Example 1: Get metadata for a list", and so on, so the reader can easily distinguish what the different examples are showing.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d0f0ac3](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/pull/3846/commits/d0f0ac3fc9eb1f2dedaac01a6183b12f49366a92)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

File: api-reference/beta/api/list-get.md [hunk 2]
```diff
@@ -129,6 +129,69 @@ Content-type: application/json
...
+        "Category": {
+          "termId": "902e568b-9b2d-4d06-87c2-2cf8ecf9f2b5" ,
+          "name": "Mechanical Device"
+         }
+       }
+    }
+  ]
+}
+```
+For more details on the ManagedMetadata resource, refer [this](ManagedMetadata.md).
```
Comment (Lauragra) [inline api-reference/beta/api/list-get.md]: This link doesn't work. What are you trying to link to here? Please change the test to "see [title of link target](url)."

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d0f0ac3](https://github.com/Vridhi-Chowdhry/microsoft-graph-docs/pull/3846/commits/d0f0ac3fc9eb1f2dedaac01a6183b12f49366a92)_: 

### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/microsoftgraph/microsoft-graph-docs, working directory: W:\y0wb-s, target branch: master, target branch commit: b06a23670cbaa39252ca32ca9a9a99c655db84c7, source branch: 803f19b8ceced4988ff21e4f7a9a45ff/patch-1, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901242301133979-3846%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9F0NchaNDMmNstdNFReat9fSjBAevjqS567JtIB6KJo%3D&st=2019-01-24T22%3A57%3A08Z&se=2019-02-24T23%3A02%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**